modules/UI/UI.js

```diff
@@ -1459,4 +1436,44 @@ UI.onUserFeaturesChanged = function (user) {
     VideoLayout.onUserFeaturesChanged(user);
 };
 
+const UIListeners = new Map([
```

Comment:
Shouldn't we try to register the listeners per feature. For example let the Toolbar register all listeners it's interested in and not trying to re-group them here in UI.js.